src/utils.js

```diff
@@ -121,7 +121,13 @@ function findNestedAggregationItem (oTargetControl, oConstraints, bDeep, aContro
         remaining: aControlsToCheck
     };
 }
-
+/**
+ * Apply test of property value against expectes value
+ *
+ * @param sOperand {string} type of comparision
+ * @param sText {string} property value provided by controls getter
+ * @param sValue {string|boolean} expected value from test
```

Comment:
would write {variant} instead of string|boolean and have a check for that value. Better to throw immediately if the utils API are called incorrectly...